src/gt4py/backend/pyext_builder.py

```diff
@@ -122,6 +122,9 @@ def get_gt_pyext_build_opts(
         extra_compile_args["cxx"].append(
             "-isystem{}".format(os.path.join(dace_path, "runtime/include"))
         )
+        extra_compile_args["nvcc"].append(
```

Comment:
This could be combined with the append above to make the intent cleaner and the type check a little cleaner. Not critical though.
```python
# I assume this returns an Optional[str]? Would be better to check `is not None to avoid issues with empty string.
if dace_path := get_dace_module_path():
    comp_arg = "-isystem{}".format(os.path.join(dace_path, "runtime/include"))
    extra_compile_args["cxx"].append(comp_arg)
    extra_compile_args["nvcc"].append(comp_arg)
```

Reply:
They are different though: `-isystem{}` for `cxx` but `-isystem={}` for nvcc (I.e. without and with `=`)

Reply:
oh odd! I didn't realize that. Is that a property of the compiler?

Reply:
I think so. At least the options above follow the same pattern and nvcc seems not to support the version without `=`.